percona-monitoring-and-management/secret.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pmm-secret1
+  namespace: default
+type: Opaque
+data:
+  PMM_ADMIN_PASSWORD: $PMM_PASSWORD
```

Comment:
`PMM_ADMIN_PASSWORD: $PMM_PASSWORD` should be
`PMM_ADMIN_PASSWORD: $PMM_ADMIN_PASSWORD`

you have env:
```
configuration:
  PMM_ADMIN_PASSWORD:
```

Reply:
We need to encode to base64 first
```PMM_PASSWORD=$(echo ${PMM_ADMIN_PASSWORD}|base64)
```